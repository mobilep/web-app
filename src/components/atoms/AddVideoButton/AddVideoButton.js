import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import { MessageBoxBaseButton } from '../..';
import { VideoCameraIcon } from '../../../icons';

import './styles.css';

const getClassName = (className) => classNames('AddVideoButton', { [className]: className });

const AddVideoButton = ({ className, ...rest }) => (
	<MessageBoxBaseButton { ...rest } className={getClassName(className)} >
		<VideoCameraIcon />
	</MessageBoxBaseButton>
);

AddVideoButton.propTypes = {
	className: PropTypes.string,
};

export default AddVideoButton;
