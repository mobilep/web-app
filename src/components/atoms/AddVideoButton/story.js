import React from 'react';
import { storiesOf } from '@storybook/react';
import AddVideoButton from './AddVideoButton';

storiesOf('1. Atoms / AddVideoButton', module)
	.add('AddVideoButton', () => <AddVideoButton />);
