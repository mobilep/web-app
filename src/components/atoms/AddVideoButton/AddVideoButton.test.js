import React from 'react';
import { shallow } from 'enzyme';

import AddVideoButton from './AddVideoButton';

describe('Atom - AddVideoButton', () => {
	it('Should render', () => {
		const wrapper = shallow(<AddVideoButton />);
		expect(wrapper).toMatchSnapshot();
	});
});
