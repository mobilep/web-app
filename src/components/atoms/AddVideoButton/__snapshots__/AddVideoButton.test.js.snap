// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - AddVideoButton Should render 1`] = `
<MessageBoxBaseButton
  className="AddVideoButton"
>
  <VideoCameraIcon />
</MessageBoxBaseButton>
`;
